modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -283,6 +283,13 @@ export class {{classname}} {
             localVarHttpContext = new HttpContext();
         }
 {{/httpContextInOptions}}
+{{#httpTransferCacheInOptions}}
+
+        let localVarTransferCache: boolean | undefined = options && options.transferCache;
+        if (localVarTransferCache === undefined) {
```

Comment:
how about using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing?

Reply:
(could be done in a separate PR, in order to unblock this one)

Reply:
I was trying to stay close to the already existing code style that's why it's a little verbose, but sure I'll open another PR for it when I can.